bpfd/src/bpf.rs

```diff
@@ -243,15 +242,25 @@ impl BpfManager {
                 Err(e)
             })?;
         self.dispatchers.insert(did, dispatcher);
+
+        // update programs with now populated kernel info
```

Comment:
@dave-tucker I'm not sure how to do this any better ATM without dramatically changing some of our dataflow/function design.... Maybe I just can't see if because I'm tired could you provide some input?

Reply:
You could get the info at load time here: https://github.com/astoycos/bpfd/blob/066e596a531d155a261be41fc0cf035fb39b2589/bpfd/src/multiprog/xdp.rs#L148